modules/js/perf/perf_imgproc/perf_cvtcolor.js

```diff
@@ -212,16 +212,14 @@ cv.onRuntimeInitialized = () => {
   const EdgeAwareBayerModeSize = [cvSize.szVGA, cvSize.sz720p, cvSize.sz1080p, cvSize.sz130x60];
   const combiEdgeAwareBayer = HelpFunc.combine(EdgeAwareBayerModeSize, EdgeAwareBayerMode);
   
-  function getConversionInfo(cvtMode)
-  {
-    switch(cvtMode)
-    {
+  function getConversionInfo(cvtMode) {
+    switch(cvtMode) {
       case "COLOR_BayerBG2GRAY": case "COLOR_BayerGB2GRAY":
       case "COLOR_BayerGR2GRAY": case "COLOR_BayerRG2GRAY":
       case "COLOR_YUV2GRAY_420":
-          return [1, 1];
+        return [1, 1];  // The returned array means the channel number of source and destination mat. 
```

Comment:
Please add the comment before the declaration `getConversionInfo`.

Reply:
You'd better mention "this function returns an array. The 1st element is the channel number of source mat and 2nd element is the channel number of destination mat."

Reply:
Got it! Thanks!